prboom2/src/s_sound.c

```diff
@@ -252,11 +255,10 @@ void S_Start(void)
   S_ChangeMusic(mnum, true);
 }
 
-void S_StartSoundAtVolume(void *origin_p, int sfx_id, int volume)
+void S_StartSoundAtVolume(degenmobj_t *origin, int sfx_id, int volume)
```

Comment:
Hm, but this feels just as wrong, as `origin` nearly always points to a `mobj_t`.